index.js

```diff
@@ -18,6 +20,7 @@ module.exports = options => through2.obj({
   if (file.isNull()) {
     return callback(null, file);
   }
+  const _log = options && options.quiet ? noop : log;
```

Comment:
How about this?

```js
const log = options && options.quiet ? () => {} : require('fancy-log');
```

Reply:
I thought about that, but then you'd end up either reimporting `fancy-log` or pointlessly redefining a a noop function for each file.  I could bump the log import up into the first function like this I suppose, which would avoid that problem, but I thought making an extra boolean check per file wasn't much overhead.

```
module.exports = options => {
  const log = options && options.quiet ? () => {} : require('fancy-log');
  return through2.obj({                                      
    maxConcurrency: options ? options.concurrent : null                           
  }, (file, enc, callback) => {
    // log() is now defined in here but doesn't need to be recreated per file
    // rest of the function snipped
  });
};                                                   
```

Reply:
If that's it, how about this.

```js
const fancyLog = require('fancy-log');

...

const log = options && options.quiet ? () => {} : fancyLog;
```

I think, redefining is not a big thing (this is slightly simpler)... But if you would like to refer `noop`, please define as following.

```js
const SUPPORTED_EXTENSIONS = ['.jpg', '.jpeg', '.png', '.gif', '.svg'];
const NOOP = () -> {};
```

Reply:
Yeah, redefining isn't a big thing, it just seemed like it was easily avoidable so I might as well avoid it (and `requrie()`ing is a bit bigger).  I'll amend with your `NOOP` suggestion.